src/Ide/Plugin/Eval.hs

```diff
@@ -87,7 +87,11 @@ import           Control.DeepSeq                ( NFData
                                                 , deepseq
                                                 )
 import Outputable (Outputable(ppr), showSDoc)
-import Control.Applicative ((<|>))
+import Data.Char (isSpace)
+import Control.Arrow (Arrow(second))
+import GHC (Ghc)
+import Type.Reflection (Typeable)
+import GHC (exprType)
```

Comment:
Let's reorganise these imports please

Reply:
Just pushed the commit that reorganises import list.
Anyway, is there any standard formatter and their configuration file in HLS, that could take care of import lists?
It seems ormolu doesn't reorganise import list.

Reply:
> > I'm not sure I understand the issue with the current module. Is it the local default declarations, or the local pragmas, that are ignored? In any case, I don't see why either would be ignored. I would leave it as a known issue.
> 
> Oops, sorry for the lack of explanation. It just ignores the `default` declaration specified on the top of the module.
> I've just added additional explanation on the top.

It might be a good idea to add a test that shows the expected behaviour and tag it as "expected failure - known issue".